chain/rand/fake.go

```diff
@@ -0,0 +1,29 @@
+package rand
```

Comment:
Can you add the description of the package goal? Randomness is a sensitive topic. And it is probably not true randomness but `fake` is invisible from the package management perspective.

Reply:
Sure! Added a brief description and a `Warn` so people are aware that this should be used with care. In our case, this is needed to fake randomness when decoupling proofs from consensus.